Source/WebCore/Modules/applepay/ApplePaySession.h

```diff
@@ -126,7 +126,7 @@ class ApplePaySession final : public PaymentSession, public ActiveDOMObject, pub
 #endif
     void didCancelPaymentSession(PaymentSessionError&&) override;
 
-    PaymentCoordinator& paymentCoordinator() const;
+    Ref<PaymentCoordinator> paymentCoordinator() const;
```

Comment:
No, this should keep returning a `PaymentCoordinator&`

Reply:
We should add a new function named `protectedPaymentCoordinator()` which returns a Ref. This is how we usually do things.